GitCommands/UserRepositoryHistory/RecentRepoInfo.cs

```diff
@@ -44,6 +48,30 @@ public RecentRepoInfo(Repository repo, bool mostRecent)
         public override string ToString() => Repo.ToString();
     }
 
+    public enum RepoType
+    {
+        Common,
```

Comment:
What is this?

Reply:
It just means that the currently opened repository does not have any relation with the current item (no pictogram).

Reply:
What about renaming:
Unrelated?
Noicon?
Other?